tools/mountsnoop.py

```diff
@@ -341,6 +374,8 @@ def print_event(mounts, umounts, cpu, data, size):
                 'mnt_ns': event.union.enter.mnt_ns,
                 'comm': event.union.enter.comm,
                 'flags': event.union.enter.flags,
+                'ppid': event.ppid if event.ppid > 0 else get_ppid(event.pid),
```

Comment:
Did you really see a case where you need to call get_ppid(event.pid) to get ppid? I though kernel found find event.ppid properly in all cases. If we missed something, we should understand and add proper bpf logic.